brayns/Brayns.cpp

```diff
@@ -58,16 +56,25 @@
 #include <plugins/DeflectPlugin/DeflectPlugin.h>
 #endif
 
-#ifdef BRAYNS_USE_BRION
+#if BRAYNS_ASSIMP_ENABLED
+#include <brayns/io/MeshLoader.h>
+#endif
+
+#if BRAYNS_USE_BRION
+#if BRAYNS_ASSIMP_ENABLED
```

Comment:
hmm, ideally the circuit loading would work w/o assimp as well, if you consider only using simple geometries and/or sdf. I guess that's not easily doable right now because of the tight coupling inside the circuit loader.

Reply:
I think I can do it with using the defines actually.